[2025-01-25T21:45:46.139+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T21:45:46.155+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [queued]>
[2025-01-25T21:45:46.165+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [queued]>
[2025-01-25T21:45:46.166+0000] {taskinstance.py:2865} INFO - Starting attempt 8 of 9
[2025-01-25T21:45:46.178+0000] {taskinstance.py:2888} INFO - Executing <Task(UploadContentsToS3): upload_contents_to_s3_task> on 2025-01-25 21:19:33.960892+00:00
[2025-01-25T21:45:46.190+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=28785) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-25T21:45:46.191+0000] {standard_task_runner.py:72} INFO - Started process 28798 to run task
[2025-01-25T21:45:46.192+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'upload_contents_to_s3_task', 'manual__2025-01-25T21:19:33.960892+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpiiyh0pb_']
[2025-01-25T21:45:46.194+0000] {standard_task_runner.py:105} INFO - Job 196: Subtask upload_contents_to_s3_task
[2025-01-25T21:45:46.232+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [running]> on host 41bcec3d818b
[2025-01-25T21:45:46.302+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='upload_contents_to_s3_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T21:19:33.960892+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T21:19:33.960892+00:00'
[2025-01-25T21:45:46.304+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T21:45:46.325+0000] {base.py:84} INFO - Retrieving connection 'minio_s3_conn_id'
[2025-01-25T21:45:46.326+0000] {connection_wrapper.py:391} INFO - AWS Connection (conn_id='minio_s3_conn_id', conn_type='aws') credentials retrieved from extra.
[2025-01-25T21:45:46.327+0000] {warnings.py:112} WARNING - <string>:9: AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead

[2025-01-25T21:45:46.549+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/ingestion_checkpoint_logs into S3 bucket diario-oficial-db-dump/ingestion_checkpoint_logs/
[2025-01-25T21:45:46.550+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/ingestion_checkpoint_logs into S3 bucket diario-oficial-db-dump/ingestion_checkpoint_logs/
[2025-01-25T21:45:46.694+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/texts into S3 bucket diario-oficial-db-dump/texts/
[2025-01-25T21:45:46.696+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/texts into S3 bucket diario-oficial-db-dump/texts/
[2025-01-25T21:45:46.848+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/preprocessed_documents/ into S3 bucket diario-oficial-db-dump/preprocessed_documents
[2025-01-25T21:45:46.851+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/preprocessed_documents/ into S3 bucket diario-oficial-db-dump/preprocessed_documents
[2025-01-25T21:45:47.983+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/vector_db/ into S3 bucket diario-oficial-db-dump/vector_db/
[2025-01-25T21:45:47.985+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/vector_db/ into S3 bucket diario-oficial-db-dump/vector_db/
[2025-01-25T21:45:48.165+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T21:45:48.166+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=upload_contents_to_s3_task, run_id=manual__2025-01-25T21:19:33.960892+00:00, execution_date=20250125T211933, start_date=20250125T214546, end_date=20250125T214548
[2025-01-25T21:45:48.213+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T21:45:48.227+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-25T21:45:48.229+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
